proto/sequencerblockapis/astria/sequencerblock/v1alpha1/optimistic_block.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+package astria.auctioneer.v1alpha1;
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
+import "astria/execution/v1alpha2/execution.proto";
+import "astria/sequencerblock/v1alpha1/block.proto";
+import "astria/primitive/v1/types.proto";
+
+// The "BaseBlock" is the information needed to simulate bundles on top of
+// a Sequencer block which may not have been committed yet.
+message BaseBlock {
+    // This is the block hash for the proposed block.
+    bytes sequencer_block_hash = 1;
+    // The hash of previous rollup block, which new block will be created on top of.
+    bytes prev_rollup_block_hash = 3;
+    // List of transactions to include in the new block.
+    repeated astria.sequencerblock.v1alpha1.RollupData transactions = 4;
+    // Timestamp to be used for new block.
+    google.protobuf.Timestamp timestamp = 5;
+}
+
+// Identifying metadata for blocks that have been successfully committed in the Sequencer.
+message BlockCommit {
+    // Height of the sequencer block that was committed.
+    uint64 height = 1;
+    // Hash of the sequencer block that was committed.
+    bytes block_hash = 2;
+}
+
+message StreamOptimisticBlockRequest {
+    // The rollup id for which the Sequencer block is being streamed.
+    astria.primitive.v1.RollupId rollup_id = 1;
```

Comment:
should this potentially be an array of rollup ids (for potential multi-rollup bundling)?

Reply:
In my mind, auctioneer is very specific only to flame so i suppose having a single rollup id should be fine.

Reply:
yeah i guess this needs a bigger product discussion, like do we ever see this being used for not-flame or cross-rollup in general

Reply:
I think it definitely could make sense to do this for multiple rollups but there's not much use for it right now so can be fine to keep as a single rollup id for `v1alpha1`